homeassistant/components/iss/__init__.py

```diff
@@ -1,18 +1,69 @@
 """The iss component."""
 from __future__ import annotations
 
+from datetime import datetime, timedelta
+import logging
+from typing import TypedDict
+
+import pyiss
+import requests
+from requests.exceptions import HTTPError
+
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
 from .const import DOMAIN
 
+_LOGGER = logging.getLogger(__name__)
+
 PLATFORMS = [Platform.BINARY_SENSOR]
 
 
+class IssData(TypedDict):
```

Comment:
Please use a dataclass instead. That's more developer friendly.

https://docs.python.org/3/library/dataclasses.html

Reply:
Still new to python, so hope I have done it correctly